cloverage/src/cloverage/coverage.clj

```diff
@@ -269,6 +269,9 @@
             (mark-loaded namespace)))
 
         (println "Instrumented namespaces.")
+        ;; load runner multimethod definition from other dependencies
+        (when (io/resource (format "%s/cloverage.clj" (name runner)))
+          (load (format "/%s/cloverage.clj" (name runner))))
```

Comment:
The existence of this feature should probably be documented.

Reply:
I agree, but it might be better to wait till we have a reference implementation of an external runner to point to.

Reply:
Echoing @ataggart here. If CircleCI is willing to commit to an open source external runner and the relevant docs (in this repo), I'd be happy to usher this along for now, but this definitely needs some documentation eventually :)

Perhaps a nit that might be resolved if docs are clearer, but perhaps the name could be a little more specific too. Also, note to self: my first concern was "but what about resource namespace clashes -- this gets to execute arbitrary code!" -- that's true, but you have to opt in to a runner when you run Cloverage, and anyone who gets to write arbitrary resources probably has a ton of code exec already.